sdks/wasm/Makefile

```diff
@@ -57,26 +60,27 @@ toolchain: .stamp-v8
 driver.o: driver.c
 	$(EMCC) -g -Os -s WASM=1 -s ALLOW_MEMORY_GROWTH=1 -s BINARYEN=1 -s "BINARYEN_TRAP_MODE='clamp'" -s TOTAL_MEMORY=134217728 -s ALIASING_FUNCTION_POINTERS=0 driver.c -c -o driver.o
 
+#SAFE_HEAP=1
 mono.js: driver.o library_mono.js $(TOP)/sdks/out/wasm-interp/lib/libmonosgen-2.0.a
-	$(EMCC) -g4 -Os -s WASM=1 -s ALLOW_MEMORY_GROWTH=1 -s BINARYEN=1 -s "BINARYEN_TRAP_MODE='clamp'" -s TOTAL_MEMORY=134217728 -s ALIASING_FUNCTION_POINTERS=0 --js-library library_mono.js driver.o $(TOP)/sdks/out/wasm-interp/lib/libmonosgen-2.0.a -o mono.js
+	$(EMCC) -g4 -Os -s WASM=1 -s ALLOW_MEMORY_GROWTH=1 -s BINARYEN=1 -s "BINARYEN_TRAP_MODE='clamp'" -s TOTAL_MEMORY=134217728 -s ALIASING_FUNCTION_POINTERS=0 -s ASSERTIONS=2 --js-library library_mono.js driver.o $(TOP)/sdks/out/wasm-interp/lib/libmonosgen-2.0.a -o mono.js
 
 build-native: mono.js
 
 #FIXME some Make magic here would be appreciated
-.stamp-copy-bcl: $(TOP)/sdks/wasm/managed $(BCL_INPUT_ASM)
+.stamp-copy-bcl: $(BCL_INPUT_ASM)
 	mkdir -p managed/
 	cp $(BCL_INPUT_ASM) managed/
 	touch $@
 
-managed/main.exe: main.cs .stamp-copy-bcl managed/nunitlite.dll
-	mcs /nostdlib /unsafe main.cs -r:managed/nunitlite.dll $(MINI_TEST_DEPS) -out:managed/main.exe
+managed/main.exe: .stamp-copy-bcl managed/nunitlite.dll $(APP_SOURCES)
+	mcs /nostdlib /unsafe $(APP_SOURCES) -r:managed/nunitlite.dll $(MINI_TEST_DEPS) -out:managed/main.exe
```

Comment:
You should replace `-out:managed/main.exe` by `out:$@`

Reply:
fixed.